compile.sh

```diff
@@ -5,3 +5,5 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+dotnet build -o ./csharp/code ./csharp/code.csproj
```

Comment:
Shouldn't this `publish` instead of `build`?

Reply:
Good catch

Reply:
I'm not 100% sure about that - I recall an old discussion about this, but that might have been about `dotnet pack` perhaps?

Reply:
Ah, that's been fixed as of .NET 8 so this will indeed use Release.